src/main/java/at/hannibal2/skyhanni/features/inventory/QuiverNotification.kt

```diff
@@ -0,0 +1,36 @@
+package at.hannibal2.skyhanni.features.inventory
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.TitleManager
+import at.hannibal2.skyhanni.events.LorenzChatEvent
+import at.hannibal2.skyhanni.utils.SoundUtils
+import at.hannibal2.skyhanni.utils.SoundUtils.playSound
+import at.hannibal2.skyhanni.utils.StringUtils.removeColor
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import kotlin.time.Duration.Companion.seconds
+
+object QuiverNotification {
+    @SubscribeEvent
+    fun onChatMessage(event: LorenzChatEvent) {
+        if (!SkyHanniMod.configManager.features.inventory.quiverAlert) return
+        val message = event.message.removeColor()
+        if (message.startsWith("You only have") && message.endsWith("arrows left in your Quiver!")) {
+            val number = message.split(" ")[3]
```

Comment:
please have this be a pattern (repo pattern) instead of performing multiple string

Reply:
I hope i implemented it properly